### PR TITLE
Solar Abundance script

### PR DESCRIPTION
**Pull request by profxj**
Simple script to show a solar abundance value.

Docs and test too.

### CODE REVIEW

File: linetools/scripts/lt_solabnd.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+"""
+Plot the solar abundance data for an element or all elements
```
Comment (jessicawerk) [inline linetools/scripts/lt_solabnd.py]: Don't you mean 'print the solar abundance data'?

File: linetools/scripts/lt_solabnd.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+try:
+    ustr = unicode
+except NameError:
+    ustr = str
+
+def parser(options=None):
+    import argparse
+    # Parse
+    parser = argparse.ArgumentParser(
+        description='Print spectral line data of a line or lines.')
```
Comment (jessicawerk) [inline linetools/scripts/lt_solabnd.py]: Also, the description is a bit off here.

**Comment by jessicawerk** [comment]
Looks great to me aside from minor description issues noted in my comments!

File: linetools/scripts/lt_solabnd.py (hunk 1 above)
Comment (profxj) [inline linetools/scripts/lt_solabnd.py]: yup, fixed

**Comment by profxj** [comment]
Merging

**Pull request merged**